components/schema-migrator/Makefile

```
@@ -4,12 +4,16 @@ TAG = $(DOCKER_TAG)
 
 .PHONY: ci-pr ci-master ci-release build-image push-image
 
-ci-pr: build-image push-image
-ci-master: build-image push-image
-ci-release: build-image push-image
+ci-pr: build-image validate push-image
+ci-master: build-image validate push-image
+ci-release: build-image validate push-image
 
 build-image:
 	docker build -t $(IMG_NAME) .
+
+validate:
```

Maybe build-image could be a dependency of validate?
```suggestion
validate: build-image
```

---

done